### DISCUSSION TITLE
关于部署的问题

### DISCUSSION THREAD

**Discussion by woogleee** [InternLM/MindSearch#90] [Q&A]
新手小白，对这个搜索引擎蛮感兴趣，也按照Readme完成部署，（Windows+VSCODE+UBUNTU），在运行
```
npm start

```
时，似乎连不上网络，一直转转转
可以帮忙排查一下是什么原因嘛？

**Comment by woogleee** [comment]
@Harold-lkk

**Comment by Harold-lkk** [comment]
提供一下截图？

**Reply by woogleee** [reply]
![微信截图_20240805160148](https://github.com/user-attachments/assets/76a5647e-ab3f-40e6-8179-373c4ebb6756)

**Comment by Harold-lkk** [comment]
启动 app 的日志呢，大概率是因为后端服务请求 ping 不通

**Reply by woogleee** [reply]
![2](https://github.com/user-attachments/assets/5ee8d566-c418-4bf6-8e73-8d31dc2668f4)
您帮忙看看，你说的日志是这个嘛？这是最新生成的


*如果是ping不通，我应该怎么克服这个问题？*

**Comment by Harold-lkk** [comment]
启动 服务后端 的日志 运行 mindsearch.app 的那个
